### DISCUSSION TITLE
Tiles in the original game change colors if damaged

### DISCUSSION THREAD

**Discussion by XCOM-HUB** [OpenApoc/OpenApoc#1228] [General]
I remember the first time I played X-Com Apocalypse. It was a pirated copy I got from my cousins in the late 90-s from Poland. Whenever you damaged a wall or a floor tile, I remember the tile changing color to a darker shade to indicate it was compromised. Unfortunately, I no longer have that version of the game with me since I bought a Steam and a GOG version some years ago. The only problem with the Steam and GOG versions is that the tiles in the game do not change color to a darker shade when damaged (this also applies to OpenApoc). I have tried downloading another version from the internet to verify that my memory served me right but without any luck. Does anyone here remember playing a version of X-Com Apocalypse where the tiles in the Battlescape changed color to a darker shade when damaged by bullets or explosives?

Please let me know.

**Comment by makus82** [comment]
> I remember the first time I played X-Com Apocalypse. It was a pirated copy I got from my cousins in the late 90-s from Poland. Whenever you damaged a wall or a floor tile, I remember the tile changing color to a darker shade to indicate it was compromised. Unfortunately, I no longer have that version of the game with me since I bought a Steam and a GOG version some years ago. The only problem with the Steam and GOG versions is that the tiles in the game do not change color to a darker shade when damaged (this also applies to OpenApoc). I have tried downloading another version from the internet to verify that my memory served me right but without any luck. Does anyone here remember playing a version of X-Com Apocalypse where the tiles in the Battlescape changed color to a darker shade when damaged by bullets or explosives?
> 
> Please let me know.

Its can be beta2 version ! that can be be found on internet
https://www.ufopaedia.org/index.php/Cut_items_and_features_(Apocalypse)#Pre-Release_Versions
